.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "ext/cli"]
+	path = ext/cli
+	url = ../cli.git
+	branch = new_docs_site
```

Comment:
The submodule should track the latest CLI release, which isn't a branch. This field should simply be removed.

```suggestion
```

Reply:
Agreed. The plan is to remove it once the `new_docs_site` branch in the `cli` repo is merged to `main`. Then, future updates via `git submodule update --remote` will happen as part of the release trigger workflow setup (which is not part of this review and likely won't happen until the new year). That way, the submodule will only be updated when there is `cli` release. The thought is that workflow will look a bit like the one in `homebrew-cli`...where a PR and a deployment occur in the same triggered event/workflow.

Reply:
There is no reason to wait for the merge to main though... This field should never be here to begin with...

Reply:
It has been updated.